Helper/Discount.php

```diff
@@ -55,6 +55,9 @@ class Discount extends AbstractHelper
     const BOLT_DISCOUNT_CATEGORY_COUPON = 'coupon';
     const BOLT_DISCOUNT_CATEGORY_GIFTCARD = 'giftcard';
     const BOLT_DISCOUNT_CATEGORY_AUTO_PROMO = 'automatic_promotion';
+    
+    // In Magento 2, if the discount amount is less than 0.005, we can treat it as zero.
+    const ZERO_VALUE = 0.004;
```

Comment:
This is a little confusing. Why we have 0.005 in comment and 0.004 in code? Do we have corresponding code in Magento core? Could we have the same?

Reply:
@vitaliyreznikov In comment it says if less than 0.005, then we can treat it as zero. But 0.005 is not the zero value, I setup a most approximate number to represent zero value. BTW I change this zero value to 0.0049999 now.

In M2 I can not find such a const or function (@danac-gs  correct me if there is), and I test with Magento\Framework\Pricing, it works as expected.

Reply:
Why you selected 0.005? Just to understand, is it your choice or some M2 standard?

Reply:
In M2, 0.005 would be converted into 0.01 which is greater than 0 while 0.0049999 would be converted into 0

Reply:
Got it. Can we use ">=0.5" then?

Reply:
Hmm so do you mean we set the constant "ZERO_VALUE" to 0.05? If so, actually it does not equal to 0